auv_sim/auv_sim_bringup/launch/inc/start_world.launch.xml

```diff
@@ -1,11 +1,12 @@
 <?xml version="1.0"?>
 <launch>
-  <arg name="gui" default="true" />
   <arg name="debug" default="false" />
-  <arg name="world_name" default="$(find auv_sim_description)/worlds/seabed.world" />
   <arg name="namespace" default="taluy" />
-  <arg name="velocity_control" default="true" />
-  <arg name="joy_id" default="0" />
+  <arg name="world" default="pool" />
+  <arg name="gui" default="true" />
+
+  <arg name="world_name" value="$(eval find('auv_sim_description') + '/worlds/' + arg('world') + '.world')" />
+
```

Comment:
Dynamic world selection, using the world argument. roslaunch auv_sim_bringup start.launch world:=seabed